scss/_patterns_breadcrumbs.scss

```diff
@@ -1,218 +1,24 @@
-        float: left;
-        margin-left: 4px;
-      }
-
-      li {
-        color: $color-light;
-        display: inline-block;
-        margin: 0;
-        width: 100%;
-
-        @media only screen and (min-width: $breakpoint-small) {
-          display: inline-block;
-          width: auto;
-        }
-
-        // scss-lint:disable NestingDepth
-        &:first-of-type {
-          border-bottom: 1px solid $color-alto-grey;
-
-          @media only screen and (min-width: $breakpoint-small) {
-            border-bottom: 0;
-          }
-        }
-
-        a {
-          color: $color-warm-grey;
-          font-size: 1em;
-          display: inline-block;
-          width: 100%;
-          margin-right: 0;
-          padding: 8px 10px;
-          text-decoration: none;
-
-          @media only screen and (min-width: $breakpoint-small) {
-            font-size: .875em;
-          }
-        }
-
-        .breadcrumb-link,
-        .breadcrumb-link--second-level,
-        .active {
-
-          @media only screen and (min-width: $breakpoint-small) {
-            display: inline-block;
-            float: none;
-            width: auto;
-          }
-        }
-
-        .breadcrumb-link,
-        .breadcrumb-link--second-level {
-
-          &:after {
-            content: '\203A';
-            display: inline-block;
-            height: 5px;
-            margin-left: 8px;
-            position: absolute;
-            width: 5px;
-          }
-        }
-       // scss-lint:enable NestingDepth
-      }
-    }
-
-    .second-level-nav,
-    .third-level-nav {
-      display: none;
-      overflow: hidden;
-      width: 100%;
-      margin: 0;
-      padding-top: 10px;
-      padding-bottom: 10px;
-      border-bottom: 1px solid $color-alto-grey;
-
-      @media only screen and (min-width: $breakpoint-small) {
-        display: inline;
-        float: none;
-        margin-bottom: 0;
-        padding: 0;
-        width: auto;
-        border-bottom: 0;
-      }
-
-      li {
-        margin: 0;
-
-        @media only screen and (min-width: $breakpoint-small) {
-          float: none;
-          width: auto;
-        }
-
-        // scss-lint:disable NestingDepth
-        a {
-          font-size: .875em;
-          display: block;
-          width: 100%;
-          height: 100%;
-          padding: 10px;
-          color: $color-cool-grey;
-
-          @media only screen and (max-width: $breakpoint-small) {
-            &.active {
-              font-weight: 500;
-            }
-          }
-
-          @media only screen and (min-width: $breakpoint-small) {
-            padding: 8px 10px 0;
+    &__link {
+      font-weight: normal;
+      font-size: .9rem;
+      margin-right: 1rem;
+      position: relative;
 
-            &.active {
-              color: $color-highlight;
-            }
-          }
-        }
-        // scss-lint:enable NestingDepth
+      &:after {
+        content: ' ›';
```

Comment:
If we are no longer supporting second level navigation. Then I believe this can be removed as it was the leading symbol to it.